gen/definitions/virtual_network_to_fabric_site.yaml

```diff
@@ -0,0 +1,41 @@
+---
+name: Virtual Network To Fabric Site
+rest_endpoint: /dna/intent/api/v1/business/sda/virtual-network
+no_update: true
+no_import: true
+no_data_source: true
+skip_minimum_test: true
+res_description: Add virtual network (VN) to SDA Fabric Site
+doc_category: SDA
+attributes:
+  - model_name: virtualNetworkName
+    type: String
+    mandatory: true
+    data_source_query: true
+    description: Virtual Network Name, that is created at Global level
```

Comment:
This leaves me wondering. Let's say I create a VN using code below, but I have no idea whether it is "at Global level" or how to verify that:

```
  resource "catalystcenter_fabric_virtual_network" "test" {
    virtual_network_name = "SDA_VN1"
    is_guest             = false
    sg_names             = ["Employees"]
  }
```

Reply:
when you create VN with this code, you should be able to verify it here: 
/dna/provision/sd-access/virtualNetworks/list/layer_3/global

Reply:
No no, I don't mean I need explanation here. I was trying to say that `description` needs to be made more clear.